executor/prepared.go

```diff
@@ -262,7 +263,7 @@ func (e *ExecuteExec) Build() error {
 	if err != nil {
 		return errors.Trace(err)
 	}
-	b := newExecutorBuilder(e.Ctx, e.IS)
+	b := newExecutorBuilder(e.Ctx, e.IS, kv.PriorityNormal)
```

Comment:
Why use PriorityNormal here. Should we apply the same strategy with the non-prepared statement?